TickSpec/BlockParser.fs

```diff
@@ -230,6 +230,12 @@ let private parseFeatureFile parsedLines =
     | (_,_,FileStart) :: xs -> parseFeatureBlock xs
     | _ -> parsedLines |> raiseParseException "Unexpected call of parser"
 
+type internal ScannedLine =
```

Comment:
perhaps RequireQualifiedAccess and `Line.Ok` and `Line.Ignore` ?

Reply:
Sounds good!